src/ostorlab/agent/mixins/agent_open_telemetry_mixin.py

```diff
@@ -70,6 +78,13 @@ def _get_jaeger_exporter(self, parsed_url):
         logger.info('Configuring jaeger exporter..')
         return jaeger_exporter
 
+    def _get_gcp_exporter(self, parsed_url):
```

Comment:
This requires some documentation one what is expected here, what is the format of the URL, etc.